cmd/skopeo/sync.go

```diff
@@ -723,6 +726,19 @@ func (opts *syncOptions) run(args []string, stdout io.Writer) (retErr error) {
 		logrus.Warn("Running in dry-run mode")
 	}
 
+	var digestFile *os.File
+	if opts.digestFile != "" && !opts.dryRun {
+		digestFile, err = os.OpenFile(opts.digestFile, os.O_TRUNC|os.O_CREATE|os.O_WRONLY, 0644)
+		if err != nil {
+			return fmt.Errorf("Error creating digest file: %w", err)
+		}
+		defer func() {
+			if errs := digestFile.Close(); errs != nil {
+				err = fmt.Errorf("Error closing digest file: %w", err)
```

Comment:
Changing `err` in the error affects nothing AFAICS. We have `retErr` for this purpose.

Reply:
Done.